src/PrestaShopBundle/Resources/views/Admin/Product/Include/form_combinations.html.twig

```diff
@@ -148,8 +148,9 @@
                 >
                 <label class="attribute-label" for="attribute-{{ attribute.id }}">
                   <span
-                    class="pretty-checkbox {% if attribute.color is empty %} not-color {% endif %}"
+                    class="pretty-checkbox {% if attribute.color is empty and attribute.texture is empty %} not-color {% endif %}"
                     {% if attribute.color is not empty %} style="background-color: {{ attribute.color }}" {% endif %}
+                    {% if attribute.texture is not empty %} style="content: url({{ attribute.texture }})" {% endif %}
```

Comment:
[As documented in the User Guide](http://doc.prestashop.com/display/PS17/Managing+Product+Attributes):

> The next fields are only shown if the attribute is a color type.
> * **Color.** If the attribute is a color, you can enter its value in HTML color code (i.e. "#79ff52" or "lightblue"), or use the color picker to precisely show the correct hue.
> * **Texture.** If your product does not use a solid color but rather a textured one (i.e. tiger stripes), you can upload a small image file that will be displayed on the product's page. Note that this will replace the HTML color from the field above. Click on the "Save" button is order to start the upload.
    You can also use this option to let the customer choose the color variety from a picture of your product rather than a color. How it is displayed on the front-end depends on the theme you are using...